winml/lib/Api/LearningModel.h

```diff
@@ -25,10 +25,6 @@ struct LearningModel : LearningModelT<LearningModel> {
       const wss::IRandomAccessStreamReference stream,
       const winml::ILearningModelOperatorProvider operator_provider);
 
-  LearningModel(
```

Comment:
why is this being removed?

Reply:
It isnt used anymore. Removing the dead code.